test/typescript-tests/testTypes.ts

```diff
@@ -2371,9 +2484,6 @@ Factory Test
   if (math.isMatrix(x)) {
     expectTypeOf(x).toMatchTypeOf<Matrix>()
   }
-  if (math.isDenseMatrix(x)) {
```

Comment:
I think it would be good to leave this `math.isDenseMatrix` test in place? Is there a special reason to remove it?

Reply:
My apologizes removed that by accident, I added it back.